spec/rubocop/cli/cli_autocorrect_spec.rb

```diff
@@ -531,7 +531,7 @@ def verify_section
      'UnneededInterpolation offense unchanged' do
     # If we change string concatenation from plus to backslash, the string
     # literal that follows must remain a string literal.
-    source = <<-'RUBY'.strip_indent
+    source = <<~'RUBY'
```

Comment:
This is a known issue of https://github.com/jruby/jruby/issues/4260.  The JRuby build matrix does not pass until JRuby 9.2.8.0 release. Therefore, it is still pending to use heredoc for `<<~'EOS'`.

Related conversation https://github.com/rubocop-hq/rubocop/pull/7173#issuecomment-505552299

Reply:
I wish I had found that before making my changes. At least I don't have to diagnose the failing build. I'll go ahead an close this for now.